File: Tasks/Task.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Rajware.Sample.Tasks;

[Table("tasks")]
public class Task
{
    [Column("id")]
    public int ID { get; set; }
    [Column("description")]
    public string Description {get;set;} = "";
    [Column("deadline")]
    public DateTime Deadline {get;set;}
    [Column("completed")]
    public bool Completed { get; set; }
}

File: Tasks/Task.vb
Option Strict On
Option Explicit On
Option Infer On

Imports System.ComponentModel.DataAnnotations.Schema

<Table("tasks")>
Public Class Task
    <Column("id")>
    Public Property ID As Integer
    <Column("description")>
    Public Property Description As String = ""
    <Column("deadline")>
    Public Property Deadline As DateTime
    <Column("completed")>
    Public Property Completed As Boolean
End Class
